pkg/apis/pipeline/v1alpha1/run_validation_test.go

```diff
@@ -43,18 +44,39 @@ func TestRun_Invalid(t *testing.T) {
 		},
 		want: apis.ErrMissingField("spec"),
 	}, {
-		name: "missing ref",
+		name: "Empty spec",
```

Comment:
=> empty ref and spec

Reply:
the test name does not match with ref and spec both being `nil`, please add one more check in the validation routine and update this test.

Reply:
The reason is, both nil and `Empty spec` are same in terms of what happens underneath. 
Actual check says, 
`if equality.Semantic.DeepEqual(rs, &amp;RunSpec{}) {		return apis.ErrMissingField("spec")` i.e. empty spec and so is the name. Please see the reply to your comment above as well, and let me know, if this is still concerning.